src/ZuluSCSI_cdrom.cpp

```diff
@@ -606,16 +602,13 @@ static void formatRawTrackInfo(const CUETrackInfo *track, uint8_t *dest)
     dest[1] = control_adr;
     dest[2] = 0x00; // "TNO", always 0?
     dest[3] = track->track_number; // "POINT", contains track number
-
-    if (track->pregap_start > 0)
-    {
-        LBA2MSFRaw(track->pregap_start, &dest[4]);
-    }
-    else
-    {
-        LBA2MSFRaw(track->data_start, &dest[4]);
-    }
-
+    // Next three are ATIME. The spec doesn't directly address how these
```

Comment:
Interesting, I was also confused by the spec not telling what to put here. I agree that imitating existing CDs makes sense.